index.js

```diff
@@ -305,252 +301,89 @@ app.get('/slack', function(req, res) {
-    url: 'https://raw.githubusercontent.com/tessel/t2-docs/master/modules.md',
-    text: '',
-    updated: null,
-    newLink: '/docs/modules'
-  },
-  cli: {
-    type: 'API',
-    clean: "Tessel 2 CLI",
-    url: 'https://raw.githubusercontent.com/tessel/t2-docs/master/cli.md',
-    text: '',
-    updated: null,
-    newlink: '/docs/cli'
-  },
-  languages: {
-    type: 'API',
-    clean: "Supported Languages",
-    url: 'https://raw.githubusercontent.com/tessel/t2-docs/master/languages.md',
-    text: '',
-    updated: null,
-    newLink: '/docs/compatibility'
-  },
-  source: {
-    type: 'API',
-    clean: "Source & Design Files",
-    url: 'https://raw.githubusercontent.com/tessel/t2-docs/master/repos.md',
-    text: '',
-    updated: null,
-    newLink: '/docs/source'
-  },
-  accelerometer: {
-    type: 'module',
-    clean: "Accelerometer",
-    url: 'https://raw.githubusercontent.com/tessel/accel-mma84/master/README.md',
-    text: '',
-    updated: null,
-    newlink: ''
-  },
-  ambient: {
-    type: 'module',
-    clean: "Ambient",
-    url: 'https://raw.githubusercontent.com/tessel/ambient-attx4/master/README.md',
-    text: '',
-    updated: null,
-    newlink: ''
-  },
-  climate: {
-    type: 'module',
-    clean: "Climate",
-    url: 'https://raw.githubusercontent.com/tessel/climate-si7020/master/README.md',
-    text: '',
-    updated: null,
-    newlink: ''
-  },
-  gps: {
-    type: 'module',
-    clean: "GPS",
-    url: 'https://raw.githubusercontent.com/tessel/gps-a2235h/master/README.md',
-    text: '',
-    updated: null,
-    newlink: ''
-  },
-  infrared: {
-    type: 'module',
-    clean: "Infrared",
-    url: 'https://raw.githubusercontent.com/tessel/ir-attx4/master/README.md',
-    text: '',
-    updated: null,
-    newlink: ''
-  },
-  relay: {
-    type: 'module',
-    clean: "Relay",
-    url: 'https://raw.githubusercontent.com/tessel/relay-mono/master/README.md',
-    text: '',
-    updated: null,
-    newlink: ''
-  },
-  rfid: {
-    type: 'module',
-    clean: "RFID",
-    url: 'https://raw.githubusercontent.com/tessel/rfid-pn532/master/README.md',
-    text: '',
-    updated: null,
-    newlink: ''
-  },
-  servo: {
-    type: 'module',
-    clean: "Servo",
-    url: 'https://raw.githubusercontent.com/tessel/servo-pca9685/master/README.md',
-    text: '',
-    updated: null,
-    newlink: ''
-  },
-
-  communicationProtocols: {
-    type: 'tutorials',
-    clean: 'Communication Protocols',
-    url: 'https://raw.githubusercontent.com/tessel/t2-docs/master/tutorials/communication-protocols.md',
-    text: '',
-    updated: null,
-    newlink: '/docs/communicationProtocols'
-  },
-  DIYModule: {
-    type: 'tutorials',
-    clean: 'Making a DIY Module',
-    url: 'https://raw.githubusercontent.com/tessel/t2-docs/master/tutorials/diy-module.md',
-    text: '',
-    updated: null,
-    newlink: '/docs/DIYModule'
-  }
-};
+// All the docs redirects
+var gitbooksBase = 'https://tessel.gitbooks.io/docs/content/';
```

Comment:
@reconbot assuming we get docs.tessel.io per https://github.com/tessel/t2-docs/pull/51#issuecomment-233782693, we will need to change this line